js_modules/dagit/packages/core/src/ui/Filters/FilterDropdown.tsx

```diff
@@ -261,6 +348,27 @@ const TextInputWrapper = styled.div`
   }
 `;
 
+type MenuItemProps = React.ComponentProps<typeof _MenuItem> & {
+  menuKey: string;
+  index: number;
+};
+const MenuItem = React.memo(({menuKey, index, ...rest}: MenuItemProps) => {
+  const divRef = React.useRef<HTMLDivElement | null>(null);
+  const isFocused = useRecoilValue(
+    isFocusedSelector(React.useMemo(() => ({key: menuKey, index}), [index, menuKey])),
+  );
+  React.useLayoutEffect(() => {
+    if (isFocused) {
+      divRef.current?.querySelector('a')?.focus();
+    }
+  }, [isFocused]);
```

Comment:
Is it possible that there could be a search input that should retain focus even as the keyboard is used to navigate up/down to highlight results?

Reply:
I wanted focus to move with the active element. Otherwise the element that's active isn't the element that's focused and I don't think screen readers would pick that up.